src/r/DESCRIPTION

```diff
@@ -7,10 +7,10 @@ Authors@R: c(person("Hongyuan", "Jia", email = "hongyuan.jia@bears-berkeley.sg",
              person("David", "Meyer", role = "aut", comment = c(ORCID = "0000-0002-7071-7547")),
              person("Didier", "Thevenard", role = "aut", comment = c(ORCID = "0000-0002-0749-6841")))
 Description:
-    R implementation of 'PsychroLib'
+    Implementation of 'PsychroLib'
```

Comment:
```suggestion
    'PsychroLib'
```
I would remove Implementation of altogether. It's clear enough what it is.

Reply:
Agree.

Reply:
Actually could you please change it to -- sort of lost its meaning now:
```
'PsychroLib' <https://github.com/psychrometrics/psychrolib> is a library of functions to enable the calculation properties of moist and dry air in both metric (SI) and imperial (IP) systems of units. References: Meyer, D. and Thevenard, D (2019) <doi.org/10.21105/joss.01137>.

```

please check if you need to break the text before committing it to `src/r/DESCRIPTION`.

Reply:
Sure.